src/cli.ts

```diff
@@ -1,7 +1,66 @@
-function main(packageRootDir: string) {
+import meow from 'meow';
+import * as fs from 'fs';
+import pify from 'pify'
+
+import {generatePackageTree} from './package-tree';
+import {outputToUser} from './output';
+
+const cli = meow({
+  help:`  
+    Usage
+      $ TODO <project>
+ 
+    Options
+      --help        Prints this help message.
+  `,
+  flags: {
+    help: {type: 'boolean'}
+  }
+});
+
+if(cli.input.length !== 1){
```

Comment:
Some formatting issues in this file. Can you run clang format?

Reply:
Specifically needs to be a space after the `if`, and before the `{`. Then below around the `else`.

Reply:
done